capture_device: 2
capture_fps: 15.0
capture_width: 854
capture_height: 480
frame: 'camera1'
